src/registrar/templates/domain_detail.html

```diff
@@ -6,7 +6,7 @@
   <div class="margin-top-4 tablet:grid-col-10">
 
     <div
-        class="usa-summary-box dotgov-status-box margin-top-3 padding-left-2{% if domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED%} dotgov-status-box--action-need{% endif %}"
+        class="usa-summary-box dotgov-status-box margin-top-3 padding-left-2{% if not domain.is_expired %}{% if domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED %} dotgov-status-box--action-need{% endif %}{% endif %}"
```

Comment:
```suggestion
        class="usa-summary-box dotgov-status-box margin-top-3 padding-left-2{% if not domain.is_expired and (domain.state == domain.State.UNKNOWN or domain.state == domain.State.DNS_NEEDED) %} dotgov-status-box--action-need{% endif %}"
```
(Weird spacing in the editor - don't directly commit that)
(Q) Why not include this in the original if statement?

Reply:
This doesn't work in django templates.  Have to use nested ifs.